app/services/claim_file_builder/render_to_file.rb

```diff
@@ -6,10 +6,14 @@ module RenderToFile
 
       def render_to_file(object:)
         Tempfile.new.tap do |file|
-          file.write render(object)
+          file.write with_windows_lf(render(object))
           file.rewind
         end
       end
+
+      def with_windows_lf(string)
+        string.gsub(/\n/, "\r\n")
```

Comment:
An alternative would be to use the String [#encode](https://ruby-doc.org/core-2.3.3/String.html#method-i-encode) method.

e.g.
`string.encode(crlf_newline: true)`

Reply:
Nice one - never knew that one existed